business/services/ConfigurationService.go

```diff
@@ -0,0 +1,40 @@
+package services
```

Comment:
The package comment is missing, you will get go-lint warning later

Reply:
fix applied.